source/FrameBuffer.cpp

```diff
@@ -0,0 +1,90 @@
+/* FrameBuffer.cpp
+Copyright (c) 2023 by RisingLeaf
+
+Endless Sky is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free Software
+Foundation, either version 3 of the License, or (at your option) any later version.
+
+Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+this program. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#include "FrameBuffer.h"
+
+#include <map>
+
+
+
+namespace {
+	std::map<std::string, int> textureStorage;
```

Comment:
Consumers of FrameBuffer.h can just hold onto the framebuffers themselves, no need for an expensive std::map.